docs/fast_data/monitoring/dashboards/single_views.md

```diff
@@ -4,24 +4,39 @@ title: Single Views
 sidebar_label: Single Views
 ---
 
-This Dashboard is mainly concerned with Single Views related concepts, and a Single View Creator service is required for exposing the metrics of interest.
+This Dashboard is mainly concerned with Single Views related concepts and as such, it needs the Single View Creator and/or Single View Trigger Generator to expose the metrics it displays.
+
+:::caution
+Make sure the services you need the metrics from have the "Enable metric gathering" flag enabled in the console.
+:::
 
 Click <a download target="_blank" href="/docs_files_to_download/single-view-creator.json">**here**</a> to download the dashboard.
 
+### Single View Trigger Generator Service
+
+The Single View Trigger Generator Service is one of the core services of the Fast Data which replaces the Real Time Updater on the strategy execution side of things. 
+Among many advantages it also provides some metrics to help you monitor the strategy execution times.
```

Comment:
Which are these advantages?

Reply:
Well that's what the page of the SVTG is for hahaha If you want I can put a link and say "among other advantages"

Reply:
Done, wdyt?

Reply:
yes, it was a rhetorical question. You don't write Among many advantages without explaining or linking the documentation page where these advantages are explained 😄

Reply:
ok now it is better, let's move the SVTG section below the SVC

Reply:
Hmmm, are you sure? I order of flow the strategies come first and then the aggregation. Why do you prefer to invert it?

Reply:
Because we are talking about monitoring, I'm not sure the flow counts. In the end SVC is way more widespread wrt SVTG

Reply:
fair, moving it